src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/ParameterNumber/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="ParameterNumber"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="checkNameForContextStrategyByTokenOrParent"
```

Comment:
`checkNamesForContextStrategyByTokenOrParent`

To signal it is collection

Reply:
done